### ISSUE TITLE
Βήμα 18: Bagging

### ISSUE THREAD

**Issue by michaelbzms** [slp-ntua/patrec-labs#17]
Στο bagging πόσο έλεγχο έχουμε ως προς τα σύνολα που εκπαιδεύουμε κάθε instance του ίδιου ταξινομητή;

Γιατί θα θεωρούσα σκόπιμο να εξασφαλίσουμε πχ ότι σε κάθε ένα έχουμε παρόμιο αριθμό δειγμάτων από κάθε κατηγορία (πχ όπως υπάρχει stratified k-fold), αλλά δεν βλέπω κάποια σχετική παράμετρο στην sklearn που να το κάνει δυνατό.

**Comment by georgepar** [comment]
Το bagging διαλέγει υποσύνολα τυχαία με επανατοποθέτηση.

Δείτε τις σχετικές αναφορές εδώ https://scikit-learn.org/stable/modules/generated/sklearn.ensemble.BaggingClassifier.html

**Comment by michaelbzms** [comment]
Το καταλαβαίνω αυτό, απλά ρωτάω αν γνωρίζετε πως θα μπορούσαμε να ελέγξουμε το τι δειγματα μπαίνουν σε κάθε υποσύνολο (δηλαδή να μην είναι εντελώς τυχαίο) και αν κάτι τέτοιο θα ήταν καλύτερο από το εντελώς τυχαίο.

Γιατί πχ για μικρά "bag sizes" μπορεί να καταλήγουμε με πολύ λίγα δείγματα για κάποιο label (ή και καθόλου) σε σχέση με άλλα.
